service-api/module/Application/src/Mock/DrivingLicense/Validator.php

```diff
@@ -10,9 +10,9 @@ class Validator implements ValidatorInterface
 {
     public function validateDrivingLicense(string $license): string
     {
-        if (str_ends_with($license, '8')) {
+        if (str_ends_with($license, 'X')) {
             return self::NO_MATCH;
-        } elseif (str_ends_with($license, '9')) {
+        } elseif (str_ends_with($license, 'Z')) {
```

Comment:
The [mock logic](https://opgtransform.atlassian.net/browse/ID-62) defines 8 and 9 as being the failure suffixes, so we should either keep this as documented or update the documentation on the ticket